release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -442,6 +442,11 @@ With this release, you can specify the location of a DNS private zone when insta
 
 For more information, see xref:../installing/installing_gcp/installation-config-parameters-gcp.adoc#installation-configuration-parameters-additional-gcp_installation-config-parameters-gcp[Additional {gcp-short} configuration parameters].
 
+[id="ocp-release-notes-installation-azure-encrypted-vnet_{context}"]
+==== Installing a cluster on {azure-full} with virtual network encryption
+
+With this release, you can install a cluster on {azure-short} using encrypted virtual networks. You must use a machine type that supports premium SSD storage. See Microsoft's documentation about link:https://learn.microsoft.com/en-us/azure/virtual-network/how-to-create-encryption?tabs=portal[Creating a virtual network with encryption] and link:https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-encryption-overview#requirements[Requirements and Limitations] for more information.
```

Comment:
seems that `You must use a machine type that supports premium SSD storage` here is not accurate. 

Installer requires machine type with capability "PremiumIO" which is set to be true, else pre-check could not be passed and exit with error. This is also described in [installation doc ](https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html-single/installing_on_azure/index#installation-minimum-resource-requirements_installing-azure-government-region) ( In Important warning)

Reply:
Thanks Jinyun, I thought those were the same thing 😅  I've updated the PR